src/viewer.ts

```diff
@@ -86,17 +89,46 @@ export class Viewer {
     this.scene.add(this.plane)
 
     // Lights
-    this.sunlight = new THREE.HemisphereLight()
+    this.sunlight = new THREE.HemisphereLight( 0xffffff, 0xffffff, 0.6 );
     this.light1 = new THREE.DirectionalLight()
-    this.light2 = new THREE.DirectionalLight()
 
-    this.scene.add(this.sunlight)
-    this.scene.add(this.light1)
-    this.scene.add(this.light2)
+    this.sunlight = new THREE.HemisphereLight( 0xffffff, 0xffffff, 0.6 );
+    this.sunlight.color.setHSL( 0.6, 1, 0.6 );
```

Comment:
Constants like this should all come from the "settings" object (so that they can be set in the GUI, and passed to the viewer construct), maybe @vim-sroberge can help make that change later if you have difficulty figuring it out. Could do it in a subsequent PR.